CMakeLists.txt

```diff
@@ -130,6 +146,28 @@ ENDIF()
 
 SET_PROPERTY( TARGET ${TARGETS} PROPERTY POSITION_INDEPENDENT_CODE ON )
 
+set(INCLUDE_INSTALL_DIR include/ CACHE STRING "Include install directory")
+set(LIBRARIES_INSTALL_DIR lib/ CACHE STRING "Libraries install directory")
+set(LIBRARIES lib/libClothoids_linux_static.a CACHE STRING "Libraries directory")
```

Comment:
We would need to change the name here. I don't think we should have `_linux_static` suffix in there. Its not the standard way to create a library.